docs/modules/ROOT/pages/index.adoc

```diff
@@ -323,13 +325,36 @@ The `signer_payload` data structure MUST be presented to be signed by the _<<_cr
 
 IMPORTANT: If a data hash assertion is being used, the _<<C2PA claim generator>>_ MUST also follow the process described in xref:_interaction_with_data_hash_assertion[xrefstyle=full].
 
+==== Calculating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim` value in `signer_payload`, it MUST be computed as follows:
+
+. Create a valid _<<C2PA claim>>_ data structure that includes completed references to all _<<_c2pa_assertion,C2PA assertions>>,_ *including* all *<<_identity_assertion,identity assertion>>.*
+. For the specific *<<_identity_assertion,identity assertion>>* being computed, replace the `hash` value with 32 zero-value (`0x00`) bytes.
+. Serialize the _<<C2PA claim>>_ data structure using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized _<<C2PA claim>>_ data structure.
+
+The resulting hash should be used as the `expected_claim` value.
+
+IMPORTANT: Since this algorithm replaces the `hash` value for exactly one *<<_identity_assertion,identity assertion>>,* this implies that only one *<<_identity_assertion,identity assertion>>* per _<<C2PA Manifest>>_ may provide an `expected_claim` value and that this *<<_identity_assertion,identity assertion>>* MAY NOT be referenced by any other *<<_identity_assertion,identity assertion>>.*
```

Comment:
I'm not sure I understand this note. This reminds me of, e.g., signing a real estate contract with two parties (identity asserters) + notary (claim generator); each party signs their copy of the contract (asset), with empty (zero'ed) slots for the known other party; the signed copies are reassembled and signed by the notary. Isn't that possible here: to have two co-creators of an asset add their id assertions in one claim?

Reply:
Possible different aims here WRT other identity assertions: Are we locking the _identity_ of other IA signers, or the _signer payload_ that they're signing, or both?

Reply:
Multi-signature can be resolved by witnessing either or both of identity and signer payload of other IAs.